### ISSUE TITLE
Add chip classification backend

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision-fastai-plugin#10]
Before starting this, I recommend getting the semantic segmentation plugin running locally and remotely using the Potsdam example. The `chip_classification_backend_config.py` file should have very similar content to the semantic segmentation version -- the options shouldn't be any different at this point. You will need to add a chip classification example to this repo that uses the fastai plugin. You could adapt https://github.com/azavea/raster-vision-examples#spacenet-rio-building-chip-classification. 

In the backend itself, there will be a few differences. The semantic segmentation backend stores the data as a directory of images and a directory of corresponding label images. The chip classification backend will store images in a set of folders, one for each class. Note that we are now creating debug chips in the train method. Instead of using `SegmentationItemList`, you will use `ImageList`. See https://docs.fast.ai/data_block.html for more info on creating the `DataBunch`. For the metrics, you should let the `clas_idx` default to -1. 

In PyTorch, images are stored (num_channels, height, width) as opposed to RV and Tensorflow which is (height, width, num_channels).

**Comment by lewfish** [comment]
Also, you will need to use a `cnn_learner` instead of a `unet_learner` https://docs.fast.ai/vision.learner.html#cnn_learner

**Comment by lewfish** [comment]
@adeelh For inspiration on how to write the `predict` method, especially the part where you make the `Labels` and return them, see https://github.com/azavea/raster-vision/blob/master/rastervision/backend/keras_classification/backend.py#L278-L294. 

Also, you don't need to implement these options: `train_prop`, `train_count`, `tta`, or `oversample`.

**Comment by AdeelH** [comment]
Got it. And yes, I've been looking at the Keras Classification backend. From what I can tell, we can also copy over `process_scene_data` and `process_scene_results` without any modifications.
